.azure/templates/jobs/system-tests/migration_jobs.yaml

```diff
@@ -0,0 +1,11 @@
+jobs:
+  - template: '../../steps/system_test_general.yaml'
+    parameters:
+      name: 'migration'
+      display_name: 'migration-bundle'
+      profile: 'all'
+      groups: 'migration'
+      cluster_operator_install_type: 'bundle'
+      timeout: 240
```

Comment:
do we need such a big timeout for this pipeline btw?

Reply:
nope, just copy paste :D